lib/features/children/generosity_challenge/pages/generosity_challenge_overview.dart

```diff
@@ -1,39 +1,73 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
 import 'package:givt_app/features/children/generosity_challenge/cubit/generosity_challenge_cubit.dart';
 import 'package:givt_app/features/children/generosity_challenge/utils/generosity_challenge_helper.dart';
 import 'package:givt_app/features/children/generosity_challenge/widgets/day_item_placeholder.dart';
+import 'package:givt_app/features/children/generosity_challenge/widgets/generosity_app_bar.dart';
+import 'package:givt_app/utils/app_theme.dart';
 
 class GenerosityChallengeOverview extends StatelessWidget {
   const GenerosityChallengeOverview({super.key});
 
   @override
   Widget build(BuildContext context) {
     final challenge = context.read<GenerosityChallengeCubit>();
+    final user = context.read<AuthCubit>().state.user;
     return Scaffold(
-      appBar: AppBar(
-        title: const Text('Generosity Mission'),
-        // automaticallyImplyLeading: false,
+      appBar: const GenerosityAppBar(
+        title: 'Generosity Mission',
+        leading: null,
       ),
       body: SafeArea(
-        child: GridView.builder(
-          padding: const EdgeInsets.all(20),
-          itemCount: GenerosityChallengeHelper.generosityChallengeDays,
-          shrinkWrap: true,
-          gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
-            crossAxisCount: 3,
-            childAspectRatio: 2,
-            crossAxisSpacing: 10,
-            mainAxisSpacing: 10,
-          ),
-          itemBuilder: (BuildContext context, int index) {
-            return DayItemPlaceholder(
-              isCompleted: challenge.state.days[index].isCompleted,
-              isActive: challenge.state.activeDayIndex == index,
-              dayIndex: index,
-              onPressed: challenge.activeDay,
-            );
-          },
+        child: Stack(
+          children: [
+            SvgPicture.asset(
+              'assets/images/generosity_challenge_backdrop.svg',
+              width: MediaQuery.sizeOf(context).width,
+              fit: BoxFit.fitWidth,
+            ),
+            Column(
+              children: [
+                const SizedBox(height: 24),
+                SvgPicture.asset(
+                  'assets/images/family_avatar.svg',
+                  width: 60,
+                  height: 60,
+                ),
+                const SizedBox(height: 14),
+                Text(
+                  'The ${user.lastName} Family',
```

Comment:
wondering how {user.lastName} works since generosity challenge should work for unauthenticated users?

Reply:
Oh i actually didnt think about that, but i just asked maikel and andy what do we want to do, so if it is an unauthenticated user we wot show that text yet, ill fix it tomorrow